git-sync-all

```diff
@@ -14,8 +14,8 @@ import os
 import git
 import util
 
-go = util.prompt('Are you absolutely sure? [y/N]')
-if go != 'Y' and go != 'y' and go != 'yes':
+go = util.prompt_y_n('Are you absolutely sure?')
+if not go:
 	util.warn('Ok, nothing to see here...')
```

Comment:
So i preserved the current behavior and only changed ln 17 to use the new util... but it seems to me that if the user answers "no", we don't actually stop execution? bug?

Reply:
sounds like. ☹️ Mind fixing? Thanks!

And sorry for the delay on these reviews.

Reply:
No worries -- and fixed!